ironfish/src/primitives/noteEncrypted.ts

```diff
@@ -13,6 +13,7 @@ import bufio from 'bufio'
 import { Serde } from '../serde'
 import { Note } from './note'
 
+//192 + 328 = 520
```

Comment:
I think we can remove this for now. I'm getting a little wary of adding comments that may end up out-of-date, hah